badgecheck/server/app.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, redirect, render_template, request
+import json
+
+from badgecheck.verifier import verify
+
+from utils import validate_input
```

Comment:
Feels maybe a bit pre-optimizing to put such simple validation into a separate module; should we just include a private function in the app itself?

Also, should we use an absolute import here?